README.rst

```diff
@@ -9,6 +9,9 @@ It ranges from small devices like micro-controller boards up to mobile phones.
 *qm-dfu-util* allows you to download firmware to your Intel® Quark™
 Microcontroller by using the serial interface (UART).
 
+For more information about this functionality and how to interface with the
+QMSI bootloader, please refer to https://github.com/quark-mcu/qmsi#device-firmware-update-dfu .
```

Comment:
Why not linking the guide directly?
https://github.com/quark-mcu/qmsi/blob/master/doc/dfu.rst